woocommerce/woocommerce-gutenberg-products-block
# Mini Cart block: allow left, right and center alignment

Fixes #5023 

This PR adds alignment control to the Mini Cart block. Note that we can't use `align` support here because, in the editor, Gutenberg automatically wrap blocks defined `align` support inside a `div` and use `float` for left and right alignments.

Core blocks like `paragraph`, `quote` are using the same approach as implemented in this PR as well.




### Screenshots

![image](https://user-images.githubusercontent.com/5423135/142212337-b4b1bcf3-dd25-41e5-8e29-c6281696b713.png)




### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Check out this PR, build the new assets.
2. Add Mini Cart block to a page.
3. See the new alignment control in the block toolbar.
4. Try changing the alignment, see it work in the editor.
5. The default alignment is `right`. Change it to left. Save it. See the change reflects on the front-end.
